app/models/spin.rb

```diff
@@ -193,4 +193,16 @@ def update_releases
     update(releases: releases)
     true
   end
+
+  # Refresh tags of spin
+  #
+  def refresh_tags
+    tags.delete_all
```

Comment:
1 user a has tag with namn "uno"
2 user b has also tag with name "uno" in on their spins

3. User b refresh
Tag is deleted --> Taggins are deleted --> Spins from a are no longer tagged

Reply:
No !!!  this delete only relations

Reply:
OK: I didn't realize is delete_all and not destroy_all